README.md

```diff
@@ -291,3 +291,30 @@ module_test:
   box: centos6
   options: --module-prs qpid/12
 ```
+
+## Client Testing With Docker
+
+The docker/clients directory contains setup and configuration to register clients via subscription-manager using an activation key and start katello-agent. The following are pre-requisites:
+
+ * (docker)[https://docs.docker.com/installation/]
+ * (docker-compose)[https://docs.docker.com/compose/install/]
```

Comment:
I was just suggesting the yum commands in addition to the docs.  I don't want to have to go through 2 docker docs to figure out how to install it when it's just a yum install.  We don't do that elsewhere, we give the command to install EPEL not link to the Fedora wiki.